src/sentry/web/urls.py

```diff
@@ -750,9 +750,9 @@
     ),
     # DDM
     re_path(
-        r"^ddm/",
+        r"^metrics/",
         react_page_view,
-        name="ddm",
+        name="metrics",
     ),
```

Comment:
Would add second entry until we remove the redirect from `ddm` to `metrics` in the FE.
```py
# DDM
re_path(
    r"^metrics/",
    react_page_view,
    name="metrics",
),
re_path(
    r"^ddm/",
    react_page_view,
    name="ddm",
),
```